### DISCUSSION TITLE
[HELP] How to install extension on PostgreSQL 13 via Homebrew?

### DISCUSSION THREAD

**Discussion by LarryEkohe** [pgroonga/pgroonga#517] [Q&A] [answered]
Currently when I try to `brew install pgroonga` I get the following:

```
License: PostgreSQL
==> Dependencies
Build: pkg-config ✔
Required: groonga ✔, postgresql@14 ✔
==> Analytics
```

I tried many things like copy pasting the extensions from postgresql@14 to @13, alter the some files like the `pgroonga.rb` etc. 

Nothing worked so far 😓. Upgrading to Postgresql@14 is not an option due to other dependancies I have and those don't run on @14. It has to be Postgresql@13 💦 

Any help appreciated 🙇

**Comment by askdkc** [comment, marked as answer]
This is not PGroonga’s problem, but rather a Homebrew problem. Brew used to support installing different versions, but they dropped that support a long time ago. To make things worse, they changed PostgreSQL installation paths, so now many extensions, including PostGIS, are not capable of running on PostgreSQL 15 and 16 on macOS.

**Reply by LarryEkohe** [reply]
@askdkc Thanks for your input. Didn't know that 💦 

I managed it to install it 👍 not via `homebrew`, but manually using this as [source ](https://pgroonga.github.io/install/source.html#non-windows)

**Reply by askdkc** [reply]
@LarryEkohe 

Ah, good old fashioned way.
Nicely done👍
